### ISSUE TITLE
[Bug]: tpch query hung on eks with mutil cns

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#9922]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0836dce65bd6cdd496804257ebf49b44d9dd44ad
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1357" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/3a5fdd1e-2753-449c-8063-81d3369e0d71">

log is in s3. path **s3://mo-checkin-regression/result/9912/**

**Comment by volgariver6** [comment]
https://github.com/matrixorigin/matrixone/pull/9921 is the fix, wait for it to be merged.

**Comment by volgariver6** [comment]
fixed by #9921

**Comment by aressu1985** [comment]
fixed
